src/prng/mod.rs

```diff
@@ -59,82 +59,72 @@
 //! since any bias or correlation makes the output more predictable.
 //!
 //! There is a close relationship between CSPRNGs and cryptographic ciphers.
-//! Any block cipher can be turned into a CSPRNG simply by encrypting a counter.
-//! Stream ciphers are simply a CSPRNG and a combining operation, usually XOR.
-//! This means that we can easily use any stream cipher as a CSPRNG.
+//! Any block cipher can be turned into a CSPRNG by encrypting a counter. Stream
+//! ciphers are basically a CSPRNG and a combining operation, usually XOR. This
+//! means that we can easily use any stream cipher as a CSPRNG.
 //!
 //! Rand currently provides two trustworthy CSPRNGs and two CSPRNG-like PRNGs:
 //!
-//! | name | performance | memory usage | initialisation | predictability | forward-secure |
-//! |----- |------------ | -------------|--------------- |--------------- |--------------- |
-//! | [`ChaChaRng`] | moderate | 136 bytes | fast        | secure         | no             |
-//! | [`Hc128Rng`] | fast | 4176 bytes  | slow           | secure         | no             |
-//! | [`IsaacRng`] | good | 2072 bytes  | slow           | unknown        | unknown        |
-//! | [`Isaac64Rng`] | fast | 4136 bytes | slow          | unknown        | unknown        |
+//! | name | full name |  performance | initialization | memory | predictability | backtracking resistance |
+//! |------|-----------|--------------|--------------|----------|----------------|-------------------------|
+//! | [`ChaChaRng`] | ChaCha20 | ⭐⭐⭐ | ⭐⭐⭐⭐⭐ | 136 bytes | secure | no |
```

Comment:
I don't like the stars for initialisation — they could be mis-interpreted as having some effect on quality or something.

Backtracking resistance is longer than 'forward-secure' and the term needs explaining anyway.

Reply:
I think forward secrecy is much more commonly used, so I would prefer using that term as well.